appveyor.yml

```diff
@@ -91,15 +91,19 @@ install:
             $version = $env:DVersion;
             if($version -eq "stable") {
                 $latest = (Invoke-WebRequest "http://downloads.dlang.org/releases/LATEST").toString();
-                $urls = @("http://downloads.dlang.org/releases/2.x/$($latest)/dmd.$($latest).windows.7z");
+                $urls = @("http://downloads.dlang.org/releases/2.x/$($latest)/dmd.$($latest).windows.7z",
+                          "http://ftp.digitalmars.com/dmd.$($latest).windows.7z");
             }elseif($version -eq "beta") {
                 $latest = (Invoke-WebRequest "http://downloads.dlang.org/pre-releases/LATEST").toString();
                 $latestVersion = $latest.split("-")[0].split("~")[0];
-                $urls = @("http://downloads.dlang.org/pre-releases/2.x/$($latestVersion)/dmd.$($latest).windows.7z");
+                $urls = @("http://downloads.dlang.org/pre-releases/2.x/$($latestVersion)/dmd.$($latest).windows.7z",
+                          "http://ftp.digitalmars.com/dmd.$($latest).windows.7z");
             }elseif($version -eq "nightly") {
-                $urls = @("http://nightlies.dlang.org/dmd-master-2017-05-20/dmd.master.windows.7z");
+                $latest = (Invoke-WebRequest "http://nightlies.dlang.org/$($version)/LATEST").toString();
+                $urls = @("http://nightlies.dlang.org/dmd-$($latest)/dmd.master.windows.7z");
```

Comment:
This could be any branch, e.g. newCTFE too

https://github.com/dlang/installer/blob/master/script/install.sh#L457
https://github.com/dlang/installer/blob/master/script/install.sh#L487